### PR TITLE
Display "Unknown" for empty license info in packs detail page

### LINKED ISSUE #20: Empty license info should say "Unknown"
When there's no license, the detail page should say "Unknown"

image

### PR DESCRIPTION
**Pull request by arnav-t**
## Changes
- Display "Unknown" if license info is falsy.

image
Fixes #20

### CODE REVIEW

File: app/javascript/components/buildpack/Detail.tsx [hunk 1]
```diff
@@ -104,6 +104,12 @@ class Detail extends React.Component<{match: {params: any}}, { buildpack: any }>
                                     </Card.Title>
                                     <code>pack build myapp --buildpack {buildpack.latest.namespace}/{buildpack.latest.name}@{buildpack.latest.version}</code>
                                 </div>
+                                <div>
```
Comment (elbandito) [inline app/javascript/components/buildpack/Detail.tsx:107]: @arnav-t looks like you pulled these changes from your previous PR.  Could you rebase off of main?

Comment (arnav-t) [reply app/javascript/components/buildpack/Detail.tsx]: @elbandito Done. Is this good?

**Comment by arnav-t** [comment]
Odd. What happened with this test? 🤔 
image

**Review by elbandito — APPROVED** [review]
Looks good, thanks!  Just need to figure out the test failure

**Comment by elbandito** [comment]
Must have been a bad test run.  All good after a re-run.

**Pull request merged**